crates/biome_analyze/src/lib.rs

```diff
@@ -527,21 +593,9 @@ where
             }
         }
 
-        // Emit a warning for legacy suppression syntax
-        if has_legacy && range_match(self.range, range) {
-            let signal = DiagnosticSignal::new(move || {
-                SuppressionDiagnostic::new(
-                    category!("suppressions/deprecatedSuppressionComment"),
-                    range,
-                    "Suppression is using a deprecated syntax",
-                )
-                .with_tags(DiagnosticTags::DEPRECATED_CODE)
-            });
-
-            let signal = signal
-                .with_action(|| update_suppression(self.root, token, is_leading, index, text));
-
-            (self.emit_signal)(&signal)?;
+        if is_top_level_suppression {
+            self.top_level_suppressions.expand_range(range);
```

Comment:
We might encounter multiple top level comments, so we need to expand its initial range until the end.